README.md

```diff
@@ -65,6 +65,15 @@ Only `linux-amd64-libcxx-libinstana_sensor.so` is required.
 
 ## Release History
 
+### 1.11.0 (2025-01-21)
+
+  * Add support for envoy-proxy 1.32.2 1.32.3 and 1.33.0
```

Comment:
This is irrelevant in this repo here. The Envoy tracer based on OpenTracing is only usable until 1.29.x. Please remove this. TIA
```suggestion
```